src/Components/Contact/ContactForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import { Formik, Form, Field, ErrorMessage } from 'formik'
+
+import '../FormStyles.css'
+
+const ContactForm = () => {
+  return (
+    <>
+      <Formik
+        initialValues={{
+          name: '',
+          email: '',
+          phone: '',
+          message: '',
+        }}
+        validate={(values) => {
```

Comment:
this validation should be a separated function

Reply:
Done!
I know the validation could be better, I'll work on that!